app/facades/quotes_facade.rb

```diff
@@ -0,0 +1,14 @@
+class QuotesFacade
+  def self.find_quotes(search_params)
+    data = QuoteService.get_quotes(search_params)
+    quotes = data[:results]
+    quotes&.map do |quote_data|
+      Quote.new(quote_data)
+    end
+  end
+
+  def self.results_count(search_params)
+    data = QuoteService.get_quotes(search_params)
+    data[:totalCount]
+  end
```

Comment:
These two methods are making the exact same API call which isn't super efficient. What I'd do instead is return either an array or hash with both pieces of data that you need (the total count and the quote objects) so that you only need to hit the end point once. I might look something like this:
```ruby
  def self.find_quotes(search_params)
    data = QuoteService.get_quotes(search_params)
    quotes_results = data[:results]
    quotes = quotes_results&.map do |quote_data|
      Quote.new(quote_data)
    end
    [data[:totalCount], quotes]
  end
```
and then in the controller:
```ruby
def search
    @results_count, @quotes = QuotesFacade.find_quotes(params[:query])
  end
```